gadget-container/entrypoint.sh

```diff
@@ -17,12 +17,11 @@
 set -e
 
 if [ ! -r /host/etc/os-release ] ; then
-  echo "$0 must be executed in a pod with access to the host via /host" >&2
```

Comment:
Should we nonetheless check that /host/bin exists? to check that the volume /host is correctly configured.